assets/js/modules/ads/components/PAXEmbeddedApp.js

```diff
@@ -0,0 +1,230 @@
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+/**
+ * External dependencies
+ */
+import PropTypes from 'prop-types';
+import { useInterval } from 'react-use';
+
+/**
+ * WordPress dependencies
+ */
+import { useInstanceId } from '@wordpress/compose';
+import {
+	useCallback,
+	useEffect,
+	useMemo,
+	useRef,
+	useState,
+} from '@wordpress/element';
+import { __ } from '@wordpress/i18n';
+
+/**
+ * Internal dependencies
+ */
+import Data from 'googlesitekit-data';
+import { CORE_USER } from '../../../googlesitekit/datastore/user/constants';
+import CTA from '../../../components/notifications/CTA';
+import { CORE_SITE } from '../../../googlesitekit/datastore/site/constants';
+import PreviewBlock from '../../../components/PreviewBlock';
+const { useSelect } = Data;
+
+export default function PAXEmbeddedApp( {
+	// eslint-disable-next-line no-unused-vars
+	displayMode = 'default',
+	onLaunch,
+} ) {
+	const [ launchGoogleAdsAvailable, setLaunchGoogleAdsAvailable ] = useState(
+		typeof global?.google?.ads?.integration?.integrator?.launchGoogleAds ===
+			'function'
+	);
+	const [ hasLaunchedPAXApp, setHasLaunchedPAXApp ] = useState( false );
+	const [ isLoading, setIsLoading ] = useState( true );
+	const [ launchError, setLaunchError ] = useState( undefined );
+
+	const siteName = useSelect( ( select ) =>
+		select( CORE_SITE ).getSiteName()
+	);
+	const siteURL = useSelect( ( select ) =>
+		select( CORE_SITE ).getReferenceSiteURL()
+	);
+
+	const isAdBlockerActive = useSelect( ( select ) =>
+		select( CORE_USER ).isAdBlockerActive()
+	);
+
+	const instanceID = useInstanceId( PAXEmbeddedApp, 'PAXEmbeddedApp' );
+
+	const newPromise = async () => {};
+
+	const callbackWithArgs = useCallback( ( name, callback = newPromise() ) => {
+		return async ( ...args ) => {
+			await global.console.debug( `Callback for: ${ name }`, {
+				args,
+				return: callback,
+			} );
+
+			return callback;
+		};
+	}, [] );
+
+	const paxAppRef = useRef();
+
+	const elementID = useMemo( () => {
+		return `googlesitekit-pax-embedded-app-${ instanceID }`;
+	}, [ instanceID ] );
+
+	const paxConfig = useMemo( () => {
+		return {
+			...( global?._googlesitekitPAXConfig || {} ),
+			clientConfig: {
+				contentContainer: `#${ elementID }`,
+			},
+		};
+	}, [ elementID ] );
+
+	const paxServices = useMemo( () => {
+		return {
+			authenticationService: {
+				get: callbackWithArgs( 'authenticationService:get', {
+					accessToken:
+						global?._googlesitekitPAXConfig?.authAccess
+							?.oauthTokenAccess,
+					authuser: '0',
+				} ),
+				fix: callbackWithArgs( 'authenticationService:fix', {
+					retryReady: true,
+				} ),
+			},
+			businessService: {
```

Comment:
There is a `createPaxServices` function added for this purpose https://github.com/google/site-kit-wp/blob/develop/assets/js/modules/ads/pax/services.js

`businessService`, `conversionTrackingService` and `termsAndConditionsService` can be sourced from there

Reply:
Oh, nice, I missed that 👍🏻

Thanks 🙏🏻